go/config/enclave_cli_flags.go

```diff
@@ -0,0 +1,98 @@
+		ManagementContractAddressFlag: flag.NewStringFlag(ManagementContractAddressFlag, "", FlagDescriptionMap[ManagementContractAddressFlag]),
+		LogLevelFlag:                  flag.NewIntFlag(LogLevelFlag, 0, FlagDescriptionMap[LogLevelFlag]),
+		LogPathFlag:                   flag.NewStringFlag(LogPathFlag, "", FlagDescriptionMap[LogPathFlag]),
+		EdgelessDBHostFlag:            flag.NewStringFlag(EdgelessDBHostFlag, "", FlagDescriptionMap[EdgelessDBHostFlag]),
+		SQLiteDBPathFlag:              flag.NewStringFlag(SQLiteDBPathFlag, "", FlagDescriptionMap[SQLiteDBPathFlag]),
+		MinGasPriceFlag:               flag.NewInt64Flag(MinGasPriceFlag, 0, FlagDescriptionMap[MinGasPriceFlag]),
+		MessageBusAddressFlag:         flag.NewStringFlag(MessageBusAddressFlag, "", FlagDescriptionMap[MessageBusAddressFlag]),
+		SequencerIDFlag:               flag.NewStringFlag(SequencerIDFlag, "", FlagDescriptionMap[SequencerIDFlag]),
+		MaxBatchSizeFlag:              flag.NewUint64Flag(MaxBatchSizeFlag, 0, FlagDescriptionMap[MaxBatchSizeFlag]),
+		MaxRollupSizeFlag:             flag.NewUint64Flag(MaxRollupSizeFlag, 0, FlagDescriptionMap[MaxRollupSizeFlag]),
+		L2BaseFeeFlag:                 flag.NewUint64Flag(L2BaseFeeFlag, 0, ""),
+		L2CoinbaseFlag:                flag.NewStringFlag(L2CoinbaseFlag, "", ""),
+		L2GasLimitFlag:                flag.NewUint64Flag(L2GasLimitFlag, 0, ""),
+		ObscuroGenesisFlag:            flag.NewStringFlag(ObscuroGenesisFlag, "", FlagDescriptionMap[ObscuroGenesisFlag]),
+		L1ChainIDFlag:                 flag.NewInt64Flag(L1ChainIDFlag, 0, FlagDescriptionMap[L1ChainIDFlag]),
+		ObscuroChainIDFlag:            flag.NewInt64Flag(ObscuroChainIDFlag, 0, FlagDescriptionMap[ObscuroChainIDFlag]),
+		UseInMemoryDBFlag:             flag.NewBoolFlag(UseInMemoryDBFlag, false, FlagDescriptionMap[UseInMemoryDBFlag]),
+		ProfilerEnabledFlag:           flag.NewBoolFlag(ProfilerEnabledFlag, false, FlagDescriptionMap[ProfilerEnabledFlag]),
+		DebugNamespaceEnabledFlag:     flag.NewBoolFlag(DebugNamespaceEnabledFlag, false, FlagDescriptionMap[DebugNamespaceEnabledFlag]),
+	}
+}
+
+// enclaveRestrictedFlags are the flags that the enclave can receive ONLY over the Ego signed enclave.json
+var enclaveRestrictedFlags = map[string]*flag.TenFlag{
+	L1ChainIDFlag:             flag.NewInt64Flag(L1ChainIDFlag, 0, FlagDescriptionMap[L1ChainIDFlag]),
+	ObscuroChainIDFlag:        flag.NewInt64Flag(ObscuroChainIDFlag, 0, FlagDescriptionMap[ObscuroChainIDFlag]),
+	ObscuroGenesisFlag:        flag.NewStringFlag(ObscuroGenesisFlag, "", FlagDescriptionMap[ObscuroGenesisFlag]),
+	UseInMemoryDBFlag:         flag.NewBoolFlag(UseInMemoryDBFlag, false, FlagDescriptionMap[UseInMemoryDBFlag]),
+	ProfilerEnabledFlag:       flag.NewBoolFlag(ProfilerEnabledFlag, false, FlagDescriptionMap[ProfilerEnabledFlag]),
+	DebugNamespaceEnabledFlag: flag.NewBoolFlag(DebugNamespaceEnabledFlag, false, FlagDescriptionMap[DebugNamespaceEnabledFlag]),
+}
+
+// Flag names.
+const (
```

Comment:
makes sense for the file to start with this list of constants

Reply:
Fixed !